dev-tools/kubernetes/base/elastic-agent-standalone/.env

```diff
@@ -0,0 +1,3 @@
+ES_USERNAME=elastic
+ES_PASSWORD=changeme
+ES_HOST=https://elasticsearch:9200
```

Comment:
I would suggest to put namespace as variable here

Reply:
I don't think you can use a variable from an .env file in the kustomization.yml file (which is where I set the namespace for all the resources)